res/database/Default/database_programmes.xml

```diff
@@ -35808,6 +35919,7 @@ Zawiera wiele kalamburów i [3|Full] w nagrodzonym Oscarem humorze.</pl>
 			<title>
 				<de>Herbstmolke</de>
 				<en>Autumn Whey</en>
+				<pl>Herbstmolke - Jesienne mleczko</pl>
```

Comment:
Just "Jesienne mleczko"?

Reply:
What is molke in German? 
Not enough?

Reply:
Molke is the fluid part left over when turning milk into cheese.

Reply:
Think it is serwatka. Maybe circumscribed as "cheese water".